### PR TITLE
Fix broken tests caused by django-stubs 4.2.4

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
@sobolevn PTAL.

File: tests/typecheck/test_views.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@
         pass
 
     class MyView(generics.DestroyAPIView):
-        queryset = MyModel.objects.all()
+        queryset = MyModel._default_manager.all()
```
Comment (sobolevn) [inline tests/typecheck/test_views.yml:19]:
```suggestion
        queryset = MyModel.objects.all()
```

File: tests/typecheck/test_views.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@
         pass
```
Comment (sobolevn) [inline tests/typecheck/test_views.yml:16]:
```suggestion
        field = models.IntegerField()
```

**Review by sobolevn — COMMENTED** [review]
This should also fix the issue

**Comment by brianhelba** [comment]
Ah, makes sense. Applied your suggestion.

**Comment by brianhelba** [comment]
@sobolevn I thought I understood your suggestion: since `MyModel` had no fields, it was generic and had no `.objects`.

However, it looks like the tests are actually still failing with your suggestion.

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
Thank you!

**Comment by brianhelba** [comment]
No, the tests are still failing. I just pushed the import fix to isolate the actual breakage:
> main:8: error: "Type[MyModel]" has no attribute "objects"

**Comment by sobolevn** [comment]
This looks like a bug

**Comment by intgr** [comment]
I have merged @christianbundy's fix in PR https://github.com/typeddjango/djangorestframework-stubs/pull/471.

**Pull request closed without merging**